src/components/Molecules/Task/index.stories.jsx

```diff
@@ -0,0 +1,16 @@
+import { Task } from "./index";
+
+export default { component: Task };
+
+export const Default = {
+  args: {
+    onTaskNameChange: (value) => {
+      console.log("taskname changed:" + value);
```

Comment:
```suggestion
      console.log(`taskname changed: ${value}`);
```
[テンプレートリテラル](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Template_literals)を使うとこんな感じでかけたりもします